src/main/java/sirius/biz/tycho/academy/OXOMIAcademyProvider.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.tycho.academy;
+
+import sirius.kernel.commons.Hasher;
+import sirius.kernel.commons.URLBuilder;
+import sirius.kernel.di.std.Register;
+import sirius.kernel.settings.Extension;
+import sirius.kernel.xml.StructuredNode;
+import sirius.kernel.xml.XMLCall;
+
+import javax.annotation.Nonnull;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Consumer;
+
+/**
+ * Provides {@link AcademyVideoData academy videos} by loading them from OXOMI.
+ */
+@Register
```

Comment:
Should this be bound to the framework?

Reply:
Nope, this doesn't do any harm if onboarding isn't used. Also there is not "a framework" ut actually two implementations (one for jdbc and one for mongo). Therefore we commonly don't check for a framework but rather if there is an impl for "OnboardingEngine" present...